.travis.yml

```diff
@@ -5,6 +5,7 @@ language: python
 python:
   - 3.6
   - 3.5
+  - 2.7
   # TODO(DW) fix
   # - 3.4
```

Comment:
Are 3.3 and 3.4 are passing with these changes? Would be great to have those build working too.

Reply:
https://travis-ci.org/danielsclint/partridge/builds/313217295

Reply:
Got it. No prob. This can remain TODO.